test/common/common/BUILD

```diff
@@ -306,12 +306,7 @@ envoy_cc_test(
     ],
 )
 
-envoy_cc_test(
-    name = "trie_lookup_table_test",
-    srcs = ["trie_lookup_table_test.cc"],
-    rbe_pool = "6gig",
-    deps = ["//source/common/common:trie_lookup_table_lib"],
-)
+## Removed: trie_lookup_table_test since regular Trie has been removed.
```

Comment:
I kind of like the removed comment on the lib, where the comment might help if anyone had an external dependency on that lib, but I don't see any value in leaving removal comments for tests, since nothing depends on tests.

(I suspect other reviewers would say don't even leave the comment on the lib, because searching commit history for what happened to a missing library is also an option.)